Classes/Issues/GithubClient+Issues.swift

```diff
@@ -279,23 +279,51 @@ extension GithubClient {
         }))
     }
 
+    enum CollaboratorPermission: String {
+        case admin = "admin"
+        case write = "write"
+        case read = "read"
```

Comment:
Values are unnecessary here, defaults to same as name

Reply:
😱 TIL!

Reply:
It's awesome, been around since Swift 2 I think!